src/config/testnet/config.json

```diff
@@ -4,7 +4,7 @@
   "description": "Plasma network running transfer-only MoreVP governed by LeapDAO",
   "consensus": "poa",
   "nodes": [
-    "https://testnet1.leapdao.org"
+    "wss://testnet-node1.leapdao.org:1443"
```

Comment:
Why `1443` ? Some limitations on aws side?

Reply:
443 already used for https. can't map same port to 2 protocols

Reply:
infura is doing a cool thing with `wss://rinkeby.infure.io/ws` fallback. would like to know how to replicate that.

Reply:
I don't know much about aws, but looks like https://docs.aws.amazon.com/elasticloadbalancing/latest/application/load-balancer-listeners.html#path-conditions

could do that ;)

Reply:
Maybe we have to change the node's request handler to check for / and /ws and either use http or websocket

Reply:
i gave it a try, but even in the rules i can not mix http and tcp (sentinel-ws glayed out):
<img width="1208" alt="screen shot 2019-02-10 at 11 46 45 am" src="https://user-images.githubusercontent.com/659301/52532707-99473300-2d29-11e9-9ed0-e7f6433336b4.png">

Reply:
🤔 how is tesnet-sentinel-ws configured? You can configure it as http. As websocket connect use http and upgrade the connection if websocket support is announced. But not quite sure on that 😁

Reply:
yeah, as https. but the upgrade is not supported i guess.

Reply:
So docs say it should work. Maybe we can setup a screencast and try again 😁